core/src/main/kotlin/com/codattle/core/service/FileService.kt

```diff
@@ -32,4 +32,9 @@ class FileService(private val storageService: StorageService, private val fileDa
             throw exception
         }
     }
+
+    fun deleteFile(fileId: Id<File>) {
+        storageService.delete(fileId.value)
```

Comment:
Document from collection `file` should be removed too

Reply:
Fixed